README.md

```diff
@@ -74,18 +81,23 @@ this method should set the pizza's temperature to be 300. Note: you may not use
 the variable pizza inside your method.
 
 Your Answer:
-```js
-// write code here
-```
+
+var pizza = 70.bake
```

Comment:
I would rethink what the question is asking. We are prompted to define an object literal and store it to a variable `pizza`. Starting small: 
```js
 var pizza = {

}
```
next, we need to define two properties, a `temp` and a method called `bake` that when called will set the object's `temp` property to `300`
```js
var pizza = {
  temperature: 70,
  bake: function(){
    this.temperature = 300;
  }
}
```